lib/ancestry/instance_methods.rb

```diff
@@ -263,6 +263,24 @@ def descendant_ids depth_options = {}
     end
 
     def descendant_of?(node)
+      ancestor_ids[0..-2].include?(node.id)
+    end
+
+    # Indirects
+
+    def indirect_conditions
+      self.ancestry_base_class.indirect_conditions(self)
+    end
+
+    def indirects depth_options = {}
+      self.ancestry_base_class.ordered_by_ancestry.scope_depth(depth_options, depth).where indirect_conditions
```

Comment:
Do we need `order_by_ancestry` here?

Reply:
i left that in, because one might want a certain sort order